clients/admin-ui/cypress/e2e/systems-plus.cy.ts

```diff
@@ -309,9 +309,7 @@ describe("System management with Plus features", () => {
     it("can add new systems and redirects to datamap", () => {
       cy.visit(ADD_SYSTEMS_MULTIPLE_ROUTE);
       cy.wait("@getSystemVendors");
-      cy.getByTestId("row-0").within(() => {
-        cy.get('[type="checkbox"]').check({ force: true });
-      });
+      cy.get('[type="checkbox"').check({ force: true });
```

Comment:
This change is a workaround for some weird behavior with the previous `.check()` not checking the checkbox-- none of my changes should have been affecting that to my knowledge and the checkboxes works fine manually or if you select them like this.